statements/2025-09-05 ActivityPub vs AT Protocol.md

```diff
@@ -0,0 +1,49 @@
+
+### Co-signed by the following community members:
+
+* Emelia Smith (Author, Independent Fediverse / OAuth developer) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
+* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
+* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
+* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
+* Robert W. Gehl (Author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
+* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
+* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
+* Jesse Karmani (Frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
+* Tantek Çelik (Former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
+* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
+* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
+* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
+* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) ([@jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
+* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
+* Tim Chambers (Founder of Indieweb.social) [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
+* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)
+* David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / `@retr0id@retr0.id`
+* Hong Minhee (Author of Fedify and Hollo) [@hongminhee@hollo.social](https://hollo.social/@hongminhee) / [@hongminhee.org](https://bsky.app/profile/hongminhee.org)
+* Hailey Elizabeth (AT Protocol, Bluesky Social) [@hailey.at](https://bsky.app/profile/hailey.at)
+
```

Comment:
```suggestion
* Michael Foster (The Newsmast Foundation) 
* Jaz-Michael King (Executive Director, IFTAS) [@jaz@toot.wales](https://toot.wales/@jaz) / @jaz.co.uk
* Christine Lemmer-Webber (Executive Director of Spritely, ActivityPub co-author) [@cwebber](https://social.coop/@cwebber)

```

Christine will comment later with a public statement, but has privately let me know that she wishes to sign.